packages/compass-saved-aggregations-queries/src/components/no-active-connections-modal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
This modal is for a case that was not discussed when planning for this and hence I would say not final but I am hoping to keep this until Ben is back and we have a chance to discuss this.